runtime/trappist/src/lib.rs

```diff
@@ -559,11 +559,30 @@ impl pallet_dex::Config for Runtime {
 	type MinDeposit = ConstU128<{ UNITS }>;
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+pub struct AssetRegistryBenchmarkHelper;
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_asset_registry::BenchmarkHelper<AssetId> for AssetRegistryBenchmarkHelper {
+	fn get_registered_asset() -> AssetId {
+		use sp_runtime::traits::StaticLookup;
+
+		let root = frame_system::RawOrigin::Root.into();
+		let asset_id = 1;
+		let caller = frame_benchmarking::whitelisted_caller();
+		let caller_lookup = <Runtime as frame_system::Config>::Lookup::unlookup(caller);
```

Comment:
Is the whole point of the helper to create an asset with the `frame_benchmarking::whitelisted_caller()` ?

Reply:
I put a description in the comment. But, the main point is to create an asset in `pallet_assets` and return it to `asset-registry`, because the registry cannot do it on its own.